### ISSUE TITLE
Question - How to keep AD inside the field?

### ISSUE THREAD

**Issue by coatsy35** [Stephan-S/FS22_AutoDrive#978]
More and more I am seeing vehicles which should be heading for the perimeter route around a field (I usually use AudoDrive guys routes as they are ace) and they drive straight through the perimeter route and try to go through the hedge or a wooded area to get to another route as I assuem it thinks it's quicker. Is there any tips out there for keeping the vehicle inside the field and on path? i.e. a keep inside the field polygon unless at the gate/exit type thing? or particular ways to encourage it? If not could this be a future enhancement?

**Comment by aadje93** [comment]
set restrict AI to field and unloader to field. Then it will try it utmost best to stay in the field (i have unloader no here)

![image](https://github.com/Stephan-S/FS22_AutoDrive/assets/16716901/7a5bcb4d-4534-458b-becf-676f4bc3b470)

**Comment by coatsy35** [comment]
Problem I often have is the AD vehicle will try to turn past the outer field AD route and into the hedge and then hit a tree instead of turning onto the route. Or it will think it’s quicker to go through the hedge and to the next fields perimeter AD route. I have the same settings in as you and I struggle with it daily.

**Comment by mas1701** [comment]
Yeah, AD is really good in hitting stationary objects. I wonder why they are not recognized. Seems almost as if collision detection is buggy, at least while turning.

**Comment by coatsy35** [comment]
> Yeah, AD is really good in hitting stationary objects. I wonder why they are not recognized. Seems almost as if collision detection is buggy, at least while turning.

I don’t think AD has the people with time anymore to keep developing it. There is tons wrong with it and bugs atm, but no sign of them being fixed. We are lucky with CP that development is so active.
